### PR TITLE
fix: lets ensure we register prow configs in `--ng` installs

### PR DESCRIPTION
**Pull request by jstrachan**

### CODE REVIEW

**Comment by pmuir** [comment]
/hold
/lgtm

Hold to test tide only

File: pkg/jx/cmd/controller_build.go [hunk 1]
```diff
@@ -458,7 +458,11 @@ func (o *ControllerBuildOptions) updatePipelineActivity(kubeClient kubernetes.In
...
 				}
 			}
-			logURL, err := o.generateBuildLogURL(podInterface, ns, activity, buildName, pod, location, settings, o.InitGitCredentials)
+			masker, err := kube.NewLogMasker(kubeClient, ns)
+			if err != nil {
+				log.Warnf("Failed to create LogMasker in namespace %s: %s\n", ns, err.Error())
+			}
+			logURL, err := o.generateBuildLogURL(podInterface, ns, activity, buildName, pod, location, settings, o.InitGitCredentials, masker)
 			if err != nil {
 				if o.Verbose {
```
Comment (ccojocar) [inline pkg/jx/cmd/controller_build.go]: Is it `Verbose` check needed here? I would log directly the error.

**Pull request merged**